staging/src/k8s.io/apiserver/pkg/server/filters/with_retry_after.go

```diff
@@ -62,13 +62,25 @@ type ShouldRespondWithRetryAfterFunc func() (*RetryAfterParams, bool)
 //   - 'Connection: close': tear down the TCP connection
 //
 // TODO: is there a way to merge WithWaitGroup and this filter?
-func WithRetryAfter(handler http.Handler, when ShouldRespondWithRetryAfterFunc) http.Handler {
+func WithRetryAfter(handler http.Handler, shutdownDelayDurationElapsedCh <-chan struct{}) http.Handler {
```

Comment:
nit: `WithRetryAfterOnShutdown`

Reply:
@p0lyn0mial this is part of the pick from upstream